boarman_ryan_lidar_23.ipynb

```diff
@@ -0,0 +1,523 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "61ed15a0",
+   "metadata": {},
+   "source": [
+    "# LiDAR Uncertainty at NEON Sites SOAP and SJER"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "252ff2a0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import pathlib\n",
+    "\n",
+    "import earthpy as et\n",
+    "import earthpy.plot as ep\n",
+    "import geopandas as gpd\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import seaborn as sns\n",
+    "import rasterstats as rs\n",
+    "import rioxarray as rxr\n",
+    "import xarray as xr\n",
+    "import clean\n",
```

Comment:
Since this file doesn't exist I had to comment it out to get the notebook to run.

Reply:
ahhh yes I forgot to comment this out, Good catch